core/gather/driver/target-manager.js

```diff
@@ -203,6 +209,19 @@ class TargetManager extends ProtocolEventEmitter {
 
     this._enabled = false;
     this._targetIdToTargets = new Map();
+    this._targetAttachedPayloads = [];
+  }
+
+  /**
+   * @param {*} event
+   * @param {*} listener
+   */
+  on(event, listener) {
+    super.on(event, listener);
+
+    if (event === 'protocolevent') {
+      this._targetAttachedPayloads.forEach(listener);
+    }
```

Comment:
if we stick with reemitting events, there's a handy [`newListener` event](https://nodejs.org/api/events.html#event-newlistener) for this